pkg/util/string_helpers.go

```diff
@@ -0,0 +1,20 @@
+package util
```

Comment:
please change the folder and the package name to `utils`.

I think it's better to create one file there with the name `utils.go` then using this `string_helpers`